llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -3689,6 +3689,14 @@ class VPlan {
   VPBasicBlock *getEntry() { return Entry; }
   const VPBasicBlock *getEntry() const { return Entry; }
 
+  /// Methods to support access to the middle block.
```

Comment:
Might be good to add a bit more context, e.g. something like

```suggestion
  /// Returns the 'middle' block of the plan, that is the block that selects whether to execute the scalar tail loop or the exit block from the loop latch.
```

(or something nicer worded)